integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -611,20 +612,22 @@ def test_radians(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : unary_op_df(spark, data_gen).selectExpr('radians(a)'))
 
+def is_before_jdk_9():
```

Comment:
Can we add some `java -version` samples, like:
        // Allow these formats:
        // 1.8.0_72-ea
        // 9-ea
        // 9
        // 9.0.1

Reply:
ok, will add some comments.